github/github_test.go

```diff
@@ -42,9 +31,9 @@ const (
 // setup sets up a test HTTP server along with a github.Client that is
 // configured to talk to that test server. Tests should register handlers on
 // mux which provide mock responses for the API method being tested.
```

Comment:
I would like to capture the removed comments from lines 26, 29, 32, and 73 into the description of `setup` so that it is clear what the purpose of the return values are.

Reply:
Thanks for spotting, @gmlewis. Have re-inserted in the new places they are built :)